src/ADAL.PCL/IAdalLogCallback.cs

```diff
@@ -75,6 +75,8 @@ public sealed class LoggerCallbackHandler
 
         private static IAdalLogCallback _localCallback;
 
+        public static bool DefaultLogging = true;
```

Comment:
the variable name is not descriptive. Consider renaming to UseDefaultLogging or something on those lines